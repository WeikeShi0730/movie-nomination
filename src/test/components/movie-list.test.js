import React from "react";
import { shallow } from "enzyme";
import MovieList from "../../components/movie-list/movie-list.component";

it("should render MoiveList component", () => {
  const mockMovieList = [
    {
      Title: "Test Movie Name_0",
      Year: 1024,
      imdbID: "imdbID_0",
    },
    {
      Title: "Test Movie Name_1",
      Year: 1025,
      imdbID: "imdbID_1",
    },
    {
      Title: "Test Movie Name_2",
      Year: 1026,
      imdbID: "imdbID_2",
    },
  ];
  expect(shallow(<MovieList movieList={mockMovieList} />).getElement()).toMatchSnapshot();
});
