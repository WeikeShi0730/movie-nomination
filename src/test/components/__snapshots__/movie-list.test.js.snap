// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render MoiveList component 1`] = `
<div
  className="movie-list"
>
  <h2
    className="title"
  >
    Result
  </h2>
  <Memo(Connect(Movie))
    movie={
      Object {
        "Title": "Test Movie Name_0",
        "Year": 1024,
        "imdbID": "imdbID_0",
      }
    }
    nominated={false}
    onChange={[Function]}
  />
  <Memo(Connect(Movie))
    movie={
      Object {
        "Title": "Test Movie Name_1",
        "Year": 1025,
        "imdbID": "imdbID_1",
      }
    }
    nominated={false}
    onChange={[Function]}
  />
  <Memo(Connect(Movie))
    movie={
      Object {
        "Title": "Test Movie Name_2",
        "Year": 1026,
        "imdbID": "imdbID_2",
      }
    }
    nominated={false}
    onChange={[Function]}
  />
</div>
`;
